Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/PhysicsPicking.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <Editor/Picking.h>
+
+namespace EMotionFX
+{
+    static constexpr float PickingMargin = 0.01f;
```

Comment:
minor: Since it's in a header as a global variable you want it to be `inline` and not `static` so not each translation unit including this header has its own variable.

Although I don't know if the `constexpr` makes it to be substituted at compilation time, so it wouldn't matter.

Reply:
Made inline.